src/storage/src/mem_table.rs

```diff
@@ -151,21 +150,36 @@ impl MemTable {
             }
             Entry::Occupied(mut e) => {
                 let old_op = e.get_mut();
-                self.kv_size.sub_val(old_op);
```

Comment:
Dup and move this line to each branch for error consistency. So that the `kv_size` remains correct if the caller ignored the error and keep running.